<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>774</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Waterloop</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/resources.qrc">
    <normaloff>:/images/wloop_icon_yellow_icon.png</normaloff>:/images/wloop_icon_yellow_icon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color : #1b1b1b

}</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : #1b1b1b;
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="font">
       <font>
        <family>Proxima Nova Alt Rg</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page1">
       <layout class="QGridLayout" name="load_page_layout">
        <item row="0" column="0">
         <widget class="QWidget" name="load_page" native="true">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page2">
       <layout class="QGridLayout" name="gridLayout_2" rowstretch="0">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="spacing">
         <number>4</number>
        </property>
        <item row="0" column="0">
         <widget class="QWidget" name="initializer" native="true">
          <layout class="QGridLayout" name="gridLayout_4" rowstretch="0,0,0">
           <property name="verticalSpacing">
            <number>70</number>
           </property>
           <item row="0" column="0" alignment="Qt::AlignBottom">
            <widget class="QLabel" name="logo">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>4</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="resources/resources.qrc">:/images/wloop_full_med.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0" alignment="Qt::AlignHCenter|Qt::AlignTop">
            <widget class="QPushButton" name="initialize_button">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>4</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>350</width>
               <height>100</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>350</width>
               <height>100</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Proxima Nova Rg</family>
               <pointsize>20</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">


QPushButton#initialize_button {
    color : #fed138;

background-color : #1b1b1b;

border : 4px solid #fed138;
border-radius: 5px

 }

QPushButton#initialize_button:hover {


background-color : #202020;

 }

QPushButton#initialize_button:pressed {

background-color : #151515;

 }


</string>
             </property>
             <property name="text">
              <string>INITIATE</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page3">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QWidget" name="central_placeholder" native="true">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QWidget" name="placeholder1" native="true"/>
           </item>
           <item alignment="Qt::AlignVCenter">
            <widget class="QFrame" name="buttons">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>600</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>600</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">

QFrame#buttons {
   
border : 3px solid #fed138;
border-radius : 5px;
font-size : 16pt;

width: 500px;
    
}

QFrame#buttons QLineEdit{
font-size: 16pt;
}</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item alignment="Qt::AlignVCenter">
               <widget class="QWidget" name="connection" native="true">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>500</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>500</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLineEdit{

background-color: #fed138;
font-family : &quot;Proxima Nova Rg&quot;;
color : #1b1b1b;
font-size : 14pt;
min-width: 500px;





}</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0,0,0,0,0,0,0">
                 <property name="leftMargin">
                  <number>25</number>
                 </property>
                 <property name="rightMargin">
                  <number>25</number>
                 </property>
                 <property name="verticalSpacing">
                  <number>20</number>
                 </property>
                 <item row="1" column="0" rowspan="2" colspan="3" alignment="Qt::AlignVCenter">
                  <widget class="QLabel" name="title">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="font">
                    <font>
                     <family>Proxima Nova Alt Rg</family>
                     <pointsize>20</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color : #fed138</string>
                   </property>
                   <property name="text">
                    <string>CONNECT TO POD</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <spacer name="verticalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Fixed</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="5" column="0" colspan="3" alignment="Qt::AlignHCenter">
                  <widget class="QLineEdit" name="port">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>502</width>
                     <height>75</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>Proxima Nova Rg</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <property name="inputMask">
                    <string/>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="frame">
                    <bool>false</bool>
                   </property>
                   <property name="placeholderText">
                    <string>PORT NUMBER</string>
                   </property>
                  </widget>
                 </item>
                 <item row="7" column="0" colspan="3" alignment="Qt::AlignHCenter|Qt::AlignBottom">
                  <widget class="QPushButton" name="connect_button">
                   <property name="minimumSize">
                    <size>
                     <width>300</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>300</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>Proxima Nova Rg</family>
                     <pointsize>15</pointsize>
                     <weight>75</weight>
                     <italic>true</italic>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QPushButton#connect_button {
    color : #fed138;

background-color : #1b1b1b;

border : 3px solid #fed138;

border-radius : 5px;

font : 	bold italic 
 }

QPushButton#connect_button:hover {


background-color : #202020;

 }

QPushButton#connect_button:pressed {

background-color : #151515;

 }


</string>
                   </property>
                   <property name="text">
                    <string>CONNECT</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0" colspan="3" alignment="Qt::AlignHCenter">
                  <widget class="QLineEdit" name="ip_adress">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>502</width>
                     <height>75</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>Proxima Nova Rg</family>
                     <pointsize>14</pointsize>
                     <weight>50</weight>
                     <italic>false</italic>
                     <bold>false</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="frame">
                    <bool>false</bool>
                   </property>
                   <property name="placeholderText">
                    <string>POD IP ADDRESS</string>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="0">
                  <spacer name="verticalSpacer">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Fixed</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>30</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="8" column="1">
                  <widget class="QLabel" name="connection_status">
                   <property name="font">
                    <font>
                     <family>Proxima Nova Rg</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QLabel#connection_status{
color : #fed138;
font-family : &quot;Proxima Nova Rg&quot;;
font-size : 12pt;


}</string>
                   </property>
                   <property name="text">
                    <string>STATUS: WAITING FOR INPUT</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="placeholder2" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page4">
       <property name="styleSheet">
        <string notr="true">QWidget#page4  QWidget{

background-color : #232323;

}


</string>
       </property>
       <layout class="QGridLayout" name="main_layout" rowstretch="20,1" columnstretch="4,20">
        <property name="horizontalSpacing">
         <number>10</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="1">
         <widget class="QWidget" name="dash" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>800</width>
            <height>0</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget#dash &gt; QWidget{

border-top: 3px solid #fed138;


}</string>
          </property>
          <layout class="QGridLayout" name="gaugeslayout">
           <property name="spacing">
            <number>20</number>
           </property>
           <item row="0" column="0">
            <widget class="QWidget" name="battery" native="true">
             <layout class="QGridLayout" name="battery_layout">
              <item row="0" column="0">
               <widget class="QWidget" name="widget" native="true"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QWidget" name="dlim" native="true">
             <layout class="QGridLayout" name="dlim_layout">
              <item row="0" column="0">
               <widget class="QWidget" name="widget_2" native="true"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="1" rowspan="2">
            <widget class="QWidget" name="gaugewidget" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QWidget" name="table" native="true"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QWidget" name="statusbar" native="true">
          <property name="styleSheet">
           <string notr="true">background-color : #1b1b1b</string>
          </property>
          <widget class="QSlider" name="horizontalSlider">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>991</width>
             <height>41</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color : #ffffff</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Proxima Nova Alt Rg</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar {
background-color: #1b1b1b;
}

QMenuBar::item {
color : #FED138;

}

QMenuBar::item:hover{
background-color : #2b2b2b
}

QMenuBar::item:pressed{
background-color: #000000
}</string>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QStatusBar{

background-color : #1b1b1b;
min-height : 75px;


}</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
